test/integration/asset_manager_test.rb

```diff
@@ -121,26 +107,6 @@ class RemovingAPersonImage < ActiveSupport::TestCase
 
       @person.remove_image!
     end
-
-    test 'removes the person image from the file system' do
```

Comment:
I think we can remove the call to `VirusScanHelpers.simulate_virus_scan` in the setup of these (and possibly other) tests too.

Reply:
I think it would be a good idea to go through the codebase and remove as much virus checking functionality as we can now that virus checking is being handled for non-attachment assets in Whitehall. I'll raise an issue to do that.